pages/[locale]/index.tsx

```diff
@@ -0,0 +1,90 @@
+import type { NextPage } from "next"
+import Head from "next/head"
+import { useState } from "react"
+import { useTranslation } from "next-i18next"
+import Header from "../../app/components/sections/above_fold/header/Header"
+import SectionDottedCard from "../../app/components/sections/dotted_card/SectionDottedCard"
+import SectionHamburger from "../../app/components/sections/hamburger/SectionHamburger"
+import SectionInnerRounded from "../../app/components/sections/inner_rounded/SectionInnerRounded"
+import HOME_HEAD from "../../app/constants/seo/homeHead"
+import SectionAnalytics from "../../app/components/sections/analytics/SectionAnalytics"
+import ChatBotToggle from "../../app/components/chatbot_toogle/ChatBotToogle"
+import SectionContact from "../../app/components/sections/contact/SectionContact"
+import SectionUnderlineList from "../../app/components/sections/underline_list/SectionUnderlineList"
+import languageContext from "../../app/contexts/languageContext/languageContext"
+import { HomePageProps } from "../../app/constants/types/components_props/types"
+import { getStaticPaths, makeStaticProps } from "../../lib/getStatic"
+
+interface HomeProps {
+  translation: HomePageProps
+}
+
+const Home: NextPage<HomeProps> = () => {
+  const [userLanguage, setUserLanguage] = useState<"english" | "español">("english")
+
+  const { t } = useTranslation(["homePage"])
```

Comment:
if possible, could this "t" name be more descriptive? maybe "translate" or "translation"?

Reply:
Well I wanted the same thing but it's impossible, because "t" is a method of this customHook which is from the next-i18next library, and for that I intended to create a TEXT_TRANSLATIONS object to make it readable and scalable.